### DISCUSSION TITLE
How to rename argument?

### DISCUSSION THREAD

**Discussion by masovskiy** [nextcord/nextcord#1077] [Q&A]
How to rename command argument
Command Example
/find user_id:000000000000000001
For now, just this:
/find arg:000000000000000001

Please help

**Comment by DenverCoder1** [comment]
Change `arg` to `user_id` in your code. In Python, you can name the function arguments how you like.

Show your code if you are not sure what that means.

**Comment by bmaxdk** [comment]
try 
```python
import nextcord
from nextcord.ext import commands

bot = commands.Bot(command_prefix="!")

@bot.slash_command(name="find", description="Finds a user by ID")
@nextcord.option(name="user_id", description="The user's ID")
async def find(interaction: nextcord.Interaction, user_id: int):
    await interaction.response.send_message(f"Searching for user with ID: {user_id}")

```
once you bot.run you will see /find user_id:000000000000000001

**Reply by DenverCoder1** [reply]
There is no `nextcord.option` decorator. There are two ways to document arguments in Nextcord:

Option 1: Use the docstrings to document the descriptions of the command and arguments

```py
@bot.slash_command()
async def find(interaction: nextcord.Interaction, user_id: int):
    """Finds a user by ID

    Args:
        user_id: The ID of the user to find
    """
    await interaction.send(f"Searching for user with ID: {user_id}")
```

Option 2: Describe the command in the decorator and the arguments with `nextcord.SlashOption`.

```py
@bot.slash_command(description="Finds a user by ID")
async def find(
    interaction: nextcord.Interaction,
    user_id: int = nextcord.SlashOption(description="The ID of the user to find"),
):
    await interaction.send(f"Searching for user with ID: {user_id}")
```

**Reply by bmaxdk** [reply]
Thank you for the clarification, and my apologies for the mistake.

Both of the methods you provided are correct for documenting and describing command arguments in Nextcord. Depending on your preference for inline documentation or the use of decorators, either approach should be valid.
